libs/coin-evm/docs/estimateMaxSpendable.md

```diff
@@ -0,0 +1,6 @@
+# `estimateMaxSpendable.ts`
+
+## Methods
+
+#### estimateMaxSpendable <sub><sup><sub><sup>[standard]</sup><sub></sup><sub>
+Returns the maximum amount of coin an Account can use in a basic transaction, and the maximum amount of token a TokenAccount can use in a token transaction.
```

Comment:
so the amount is returned according to the type of transaction ? maybe we should add this

standard ERC20 transfer has 0 ETH amount, but in the live Operation.amount is set to the quantity of token moved ?

Reply:
Token transactions are not "typed" per se in the live, it's inferred by the Account's type. Here if you're doing an `estimateMaxSpendable` on a TokenAccount, yes the amount returned is based on the token linked to it.